Habit Tracker/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "Habit Tracker": {
+      "commandName": "Project",
+      "workingDirectory": "C:\\Users\\harry\\source\\repos\\Habit Tracker"
```

Comment:
Note that if you have a hardcoded path that will not exist on anyone elses machine you need to make sure it is called out in the readme file as a step to update.